### DISCUSSION TITLE
v1.15.6

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#877] [Announcements]
- Updated dependencies, including two important security patches

<hr /><em>This discussion was created from the release <a href='https://github.com/Kovah/LinkAce/releases/tag/v1.15.6'>v1.15.6</a>.</em>

**Comment by martadinata666** [comment]
Is the zip release acting up again? Thanks

**Reply by Kovah** [reply]
Sorry, forgot to add the zip files. They are available now.
